### PR TITLE
HBASE-22595 Changed suppressions to full qualified class name

### PR DESCRIPTION
**Pull request by HorizonNet**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 47 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 317 | master passed |
| +1 | checkstyle | 139 | master passed |
| +1 | javadoc | 11 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 256 | the patch passed |
| +1 | checkstyle | 135 | the patch passed |
| +1 | whitespace | 1 | The patch has no whitespace issues. |
| +1 | xml | 3 | The patch has no ill-formed XML file. |
| +1 | javadoc | 12 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 12 | hbase-checkstyle in the patch passed. |
| +1 | asflicense | 13 | The patch does not generate ASF License warnings. |
| | | 1014 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-316/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/316 |
| Optional Tests |  dupname  asflicense  checkstyle  javac  javadoc  unit  xml  |
| uname | Linux e0e938f2a494 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / ed30909d27 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-316/1/testReport/ |
| Max. process+thread count | 96 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle U: hbase-checkstyle |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-316/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml [hunk 1]
```diff
@@ -36,13 +36,13 @@
...
-  <suppress checks="EmptyBlockCheck" files="TestGlobalFilter.java"/>
-  <suppress checks="EmptyBlockCheck" files="TestPathFilter.java"/>
-  <suppress checks="EqualsHashCode" files="StartcodeAgnosticServerName.java"/>
-  <suppress checks="MethodLength" files="Branch1CoprocessorMethods.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.thrift.TBoundedThreadPoolServer.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestServletFilter.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestGlobalFilter.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestPathFilter.java"/>
+  <suppress checks="EqualsHashCode" files="org.apache.hadoop.hbase.favored.StartcodeAgnosticServerName.java"/>
+  <suppress checks="MethodLength" files="org.apache.hadoop.hbase.tool.coprocessorBranch1CoprocessorMethods.java"/>
```
Comment (xcangCRM) [inline hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml]: Missing a "." here?

Comment (HorizonNet) [reply hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml]: You're right. I pushed an update.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 92 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 320 | master passed |
| +1 | checkstyle | 146 | master passed |
| +1 | javadoc | 15 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 275 | the patch passed |
| +1 | checkstyle | 137 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 2 | The patch has no ill-formed XML file. |
| +1 | javadoc | 12 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 11 | hbase-checkstyle in the patch passed. |
| +1 | asflicense | 11 | The patch does not generate ASF License warnings. |
| | | 1096 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-316/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/316 |
| Optional Tests |  dupname  asflicense  checkstyle  javac  javadoc  unit  xml  |
| uname | Linux 0812991cb46c 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / c7c6140396 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-316/2/testReport/ |
| Max. process+thread count | 76 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle U: hbase-checkstyle |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-316/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml [hunk 2]
```diff
@@ -36,13 +36,13 @@
...
-  <suppress checks="EmptyBlockCheck" files="TestServletFilter.java"/>
-  <suppress checks="EmptyBlockCheck" files="TestGlobalFilter.java"/>
-  <suppress checks="EmptyBlockCheck" files="TestPathFilter.java"/>
-  <suppress checks="EqualsHashCode" files="StartcodeAgnosticServerName.java"/>
-  <suppress checks="MethodLength" files="Branch1CoprocessorMethods.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.thrift.TBoundedThreadPoolServer.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestServletFilter.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestGlobalFilter.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestPathFilter.java"/>
+  <suppress checks="EqualsHashCode" files="org.apache.hadoop.hbase.favored.StartcodeAgnosticServerName.java"/>
```
Comment (jatsakthi) [inline hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml]: Unrelated to this jira, is the naming of "StartcodeAgnosticServerName" class correct? Like, is it intentionally not StartCodeAgnosticServerName?

**Comment by jatsakthi** [comment]
lgtm

File: hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml (hunk 2 above)
Comment (HorizonNet) [reply hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml]: Not sure if is correct, but it is consistent. This class relates to `ServerName.NON_STARTCODE`, which should be `ServerName.NON_START_CODE` if the name changes.

**Pull request merged**